httpx/_urlparse.py

```diff
@@ -263,7 +263,7 @@ def urlparse(url: str = "", **kwargs: typing.Optional[str]) -> ParseResult:
     # We also exclude '/' because it is more robust to replace it with a percent
     # encoding despite it not being a requirement of the spec.
```

Comment:
Doesn't this comment become incorrect with this change?

Reply:
Good catch, thanks.